requirements.txt

```diff
@@ -1,17 +1,21 @@
-BeautifulSoup4==4.9.0
-dispatcher==1.0
-dispatch==0.7.7
-pexpect==4.8.0
-psutil==5.6.6
-pycrypto==2.6.1
-pyserial==3.4
-usbinfo==1.1.1
-pyudev==0.21.0
-netifaces==0.10.7
-numpy==1.15.2
-PyYAML==5.3.1
-PyDispatcher==2.0.5
-dpkt==1.9.1
-jsonfield==2.0.2
-urllib3==1.24.2
-ipaddress==1.0.22
+BeautifulSoup4 == 4.9.0
+dpkt == 1.9.1
+grpcio-tools >= 1.30
+ipaddress == 1.0.22
+jsonfield == 2.0.2
+netifaces == 0.10.7
+numpy == 1.15.2
+pexpect >= 4.6.0
+protobuf ~= 3.12
+psutil == 5.7.0
+pycrypto == 2.6.1
+PyDispatcher == 2.0.5
+pyserial == 3.4
+PyYAML == 5.3.1
+urllib3 >= 1.24.2
+
+# following packages need to be installed in this order
+dispatcher == 1.0
+dispatch == 0.7.7
```

Comment:
Please add/edit following packages and list them in the following order:
requests==2.24.0
dispatcher==1.0
dispatch==1.3.2
urllib3==1.25.9

Reply:
Will do in a following PR. Do we need `requests` right now? It seems like the project doesn't use that library just yet.